docs/src/_getting-started/part4.md

```diff
@@ -0,0 +1,194 @@
+&ensp;&ensp;Transformer 'dbt-postgres' setting 'password' was set in `.env`: 'password'
+$ meltano config dbt-postgres set dbname postgres
+&ensp;&ensp;Transformer 'dbt-postgres' setting 'dbname' was set in `meltano.yml`: 'postgres'
+$ meltano config dbt-postgres set schema analytics
+&ensp;&ensp;Transformer 'dbt-postgres' setting 'schema' was set in `meltano.yml`: 'analytics'
+```
+</div>
+
+## Add our source data
+The E(t)L pipeline run already added our source data into the schema `tap_github` as table `commits`. So let's add that to dbt to work with:
+
+```bash
+mkdir transform/models/tap_github
+```
+
+Add a file called `source.yml` into this directory with the following content:
+
+```yaml
+config-version: 2
+version: 2
+sources:
+  - name: tap_github     # the name we want to reference this source by
+    schema: tap_github   # the schema the raw data was loaded into
+    tables:
+      - name: commits
+```
+
+## Add a transformed model
+Add a file called `authors.sql` to the folder `transform/models/tap_github` with the following contents:
+
+```sql
+{% raw %}
+{{
+  config(
+    materialized='table'
+  )
+}}
+
+
+with base as (select *
+from {{ source('tap_github', 'commits') }}) {% endraw %}
+
+select distinct (commit -> 'author' -> 'name') as authors from base
+```
+
+This model is configured to creating a table via the `materialized='table'` configuration. The keyword `source` is used in dbt to reference the source we just created. The actual model selects the distinct author names from the commits which are wrapped into a JSON blob.
+
+## Run the transformation process
+To create the actual table, we run the dbt model via `meltano invoke dbt-postgres:run`:
+
+<div class="termy">
+
+```console
+$ meltano invoke dbt-postgres:run
+2022-09-22T12:30:31.842691Z [info     ] Environment 'dev' is active
+12:31:01  Running with dbt=1.1.2
+12:31:02  Found 1 model, 0 tests, 0 snapshots, 0 analyses, 167 macros, 0 operations, 0 seed files, 1 source, 0 exposures, 0 metrics
+12:31:02
+12:31:03  Concurrency: 2 threads (target='dev')
+12:31:03
+12:31:03  1 of 1 START table model analytics.authors ..................................... [RUN]
+---> 100%
+12:31:03  1 of 1 OK created table model analytics.authors ................................ [SELECT 2 in 0.59s]
+12:31:03
+12:31:03  Finished running 1 table model in 1.56s.
+12:31:03
+12:31:03  Completed successfully
+12:31:04
+12:31:04  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
+#
+```
+
+</div>
+
+You can check the data inside the database. There should now be a table inside the `analytics` schema called `authors` populated with data.
+
+## Run the complete pipeline
+
+To check that everything works together as a pipeline, we clean out once more and run the whole E(t)LT pipeline. Drop the tap_github.commits and the analytics.authors tables, clear the state using `meltano state clear dev:tap-github-to-target-postgres` and run the final pipeline alltogether:
```

Comment:
Similar comment as above. It may be reasonable to have a utility which spins up a local postgres database and has some of these commands ready to go 🤔 but that can be in another iteration perhaps.

Reply:
Probably, but it's a tutorial, it's about teaching. I felt it's ok to teach people about state a bit in part 4, though I agree that it would probably also fine to wrap it away, might make adoption even higher.